react/features/video-quality/reducer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import { TOGGLE_VIDEO_QUALITY_DIALOG } from './actionTypes';
+
+const DEFAULT_STATE = {
+    isOpen: false
+};
+
+ReducerRegistry.register('features/video-quality', (state = DEFAULT_STATE, action) => {
+    switch (action.type) {
+    case TOGGLE_VIDEO_QUALITY_DIALOG:
+        return {
+            ...state,
+            isOpen: !state.isOpen

```

Comment:
chat is controls its own display in the side panel, partially as a result of the various tabs of the side panel having been moved into dialogs and chat left alone. Dialog display is controlled by DialogContainer, so I'm unsure the path you're taking right now will resolve well with the current dialog management.

Reply:
okay so for this pr i just add the button in the toolbox file